src/components/WalletButton.tsx

```diff
@@ -26,10 +46,24 @@ export const WalletButton = ({ className = "" }: WalletButtonProps) => {
 
 	const copyAddress = useCallback(async () => {
 		if (base58) {
-			await navigator.clipboard.writeText(base58);
+			try {
+				await navigator.clipboard.writeText(base58);
+				message.success({ content: "Copied to clipboard!", duration: 0.7 });
```

Comment:
Maybe we can have a MESSAGE_DURATION or something const somewhere?

Reply:
yeah, @RuvenSalamon was working on lowering the default message duration in another PR which is why I just hard-coded it for now. But I'll add a TODO to remove this value once the other PR is in.

Reply:
Never mind, the PR is merged. I'll rebase and remove this duration.